tests/unit/test_serializer.py

```diff
@@ -37,17 +37,42 @@
 def test_smil_serializer(video_record_metadata):
     """Test smil serializer."""
     rec = video_record_metadata
-    serializer = Smil(record=rec)
-    data = serializer.format()
-    root = ET.fromstring(data)
-    assert root.tag == 'smil'
-    assert len(root[1][0]) == 4
-    for child in root[1][0]:
-        assert child.tag == 'video'
-        assert 'system-bitrate' in child.attrib
-        assert 'width' in child.attrib
-        assert 'height' in child.attrib
-        assert 'src' in child.attrib
+
+    def parse_and_test(rec, subfromats_num, resolution):
```

Comment:
maybe better:
`resolution` -> `first_playlist_quality` or similar?

Reply:
it sounds better first_playlist_quality